website_supplier_list/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2016 Michael Viriyananda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    website_supplier_published = fields.Boolean(
```

Comment:
The reason is `website_published` is already used by module **website_customer**.
So if there is a partner where customers are also suppliers, they can organize themselves whether to be published in **website_customer** (_Our References_) or **website_supplier** (_Our Suppliers_) or perhaps on both.